multi-skill/src/bin/mmr_from_cc.rs

```diff
@@ -0,0 +1,122 @@
+use multi_skill::data_processing::{get_dataset_by_name, read_csv, try_write_slice_to_file};
```

Comment:
Some of my comments from `codechef_checkpoints.rs` apply to this file too, so I won't copy them here.